0_global_functions_st.R

```diff
@@ -180,3 +180,30 @@ check_valid_cfg <- function(cfg, expected_no_args = 3) {
   invisible(cfg)
 }
 
+# create paths to stress test masterdata files
+create_stressdata_masterdata_file_paths <- function(data_prep_timestamp, twodii_internal) {
+  if (!twodii_internal) {
+    stop("Currently cannot provide data files for external mode.")
```

Comment:
should we make a follow up task to decide on how to provide that data and how to handle this here?

Reply:
I would add that to the bigger refactoring instead of making it an own ticket now, since there are also a few other small things that do not work in internal mode yet for the loanbook part